components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -56,14 +58,15 @@ class AddonsManagerAdapter(
     private val addonsManagerDelegate: AddonsManagerAdapterDelegate,
     addons: List<Addon>,
     private val style: Style? = null
-) : RecyclerView.Adapter<CustomViewHolder>() {
+) : ListAdapter<Any, CustomViewHolder>(DifferCallback) {
     private val scope = CoroutineScope(Dispatchers.IO)
-    private val items: List<Any>
-    private val unsupportedAddons = ArrayList<Addon>()
+    private val unsupportedAddons = mutableSetOf<Addon>()
     private val logger = Logger("AddonsManagerAdapter")
+    @VisibleForTesting
+    internal var addonsMap: MutableMap<String, Addon> = addons.associateBy({ it.id }, { it }).toMutableMap()
```

Comment:
What's the purpose of this map? Comment could be useful.

Reply:
See https://github.com/mozilla-mobile/android-components/pull/6726#discussion_r412430175, agree a comment can help, I will added it.